website/docs/getting-started/setup/installation-guides/kubernetes/migrate-to-ee-chart.md

```diff
@@ -186,12 +187,13 @@ For reference, more information about this Helm chart can be found on [the fresh
    08:21:01,794 INFO  [org.keycloak.services] (ServerService Thread Pool -- 54) KC-SERVICES0032: Import finished successfully
    ```
 
-   Once you see lines like this, press <kbd>ctrl+c</kbd> to stop.
+   When you see lines similar to this, press <kbd>ctrl+c</kbd> to stop.
 
-5. Restart for the changes to be applied:
+5. To apply the changes, restart using the following command:
 
    ```bash
    kubectl rollout restart deployment/appsmith -n <namespace>
    ```
 
-You should have your Appsmith instance running with the new Helm chart.
```

Comment:
Can we start including a conclusion/final-state statement like this on our docs? Otherwise it feels we've stopped midway and left the readers hanging.

Reply:
Can you suggest how a user can verify the new installation?